251001 小説家になろう 作品情報 取得修正

etg-lt opened the pull request:
ページ構造の微変更が行われ
掲示板有志の対応をまとめました
https://jbbs.shitaraba.net/bbs/read.cgi/computer/44668/1716423212/218-222

etg-lt committed:
250304 小説家になろう 作品情報 取得修正

ページ構造の微変更が行われ
掲示板有志の対応をまとめました
https://jbbs.shitaraba.net/bbs/read.cgi/computer/44668/1716423212/218-222

rogenobl commented:
修正を小説フォルダに置いている人などのためにも`version: 2.1`と上げておいて欲しいです。
そうなっていると、古い2.0の設定ファイルが残っているのを忘れていても、無視した旨のメッセージが出るので気付けます。

etg-lt committed:
バージョン情報を2.1に仮指定

etg-lt commented:
つどつどお世話になっています
設定ファイル読み込み圏内に、旧設定ファイルが残っている際の検知は盲点でした
少々はばかられますが、借りにと言う事で2.1に修正

数字の方は作者采配におまかせする、という事で
close #445

etg-lt committed:
250304 ノクターン/ムーンライト/ミッドナイトノベルズ 作品情報 取得修正

小説家になろうの掲示板対応に加えて
以下の掲載サイト名対応もまとめました
https://jbbs.shitaraba.net/bbs/read.cgi/computer/44668/1716423212/226-227

etg-lt committed:
251001 小説家になろう 作品情報 取得修正

作品情報ページのタイトル情報の微変更への対応

掲示板の該当部分はこちら
https://jbbs.shitaraba.net/bbs/read.cgi/computer/44668/1716423212/381-401

etg-lt committed:
251001 ノクターン/ムーンライト/ミッドナイトノベルズ 作品情報 取得修正

gitster reviewed (commented):
動作確認しました。

今回のように些細な改行やら空白などの変更によるパーサ破壊に毎回対応するのも無駄なので、

<h1\s*class="p-infotop-title">\s*<a href=".+?">(?<title>.+?)</a>\s*</h1>

みたいにしたほうが将来的に良かありませんか？

etg-lt committed:
ハーメルン 260728 目次修正

とりあえずhtmlソース構造通りに修正
しおり表示も捌けているはず

etg-lt committed:
260728 ハーメルンR18 目次修正

260624 R18ドメイン追加 を含む修正

etg-lt committed:
260728 ハーメルンR18 目次修正

260624 R18ドメイン対応込み

etg-lt committed:
260728 ハーメルン 目次修正2

投稿日時、改稿日時タグの変更に対応

etg-lt committed:
260728 ハーメルンR18 目次修正2

投稿日時、改稿日時タグの変更に対応

etg-lt committed:
260728 ハーメルン 目次修正3

インデントの対応を修正

etg-lt committed:
260728 ハーメルンR18 目次修正3

インデントの対応を修正
